docs/manual/core-extensions/news/_index.en.md

```diff
@@ -1,17 +1,13 @@
 ---
 title: 'News/Blog extension'
-description: 'This allows news entries to be managed in the back end and output using front-end modules.'
+description: 'This allows news entries to be managed in the back end and output to the front-end using front-end modules.'
```

Comment:
Shouldn't we write "**back end**" and "**front end**" consistently without a hyphen?